app/helpers/contextualization_result_helper.rb

```diff
@@ -1,6 +1,10 @@
 module ContextualizationResultHelper
   def humanized_expectation_result_item(expectation_result)
-    %Q{<li>#{status_icon(expectation_result[:result])} #{expectation_result[:explanation]}</li>}.html_safe
+    %Q{<li>#{status_icon(expectation_result[:result])} #{sanitized expectation_result[:explanation]}</li>}.html_safe
+  end
+
```

Comment:
Perhaps we could move this method to a more generic helper

Reply:
yeah, that’s fair. Does application_helper sound good?